README.md

```diff
@@ -14,3 +15,5 @@ implementation. What changes between iterations of the loop?
 Is the asymptotic complexity of tail-recursive Fibonacci different from the
 non-tail-recursive version? Why, or why not? Add your answer, including your
 reasoning, to this markdown file.
+
+I believe the asymptotic complexity of the tail-recursive Fibonacci function is the same as the non-tail-recursive version. Both of them have a time complexity of $O(2^n)$. The tail-recursion primarily affects the space complexity, not the time complexity. The time complexity of both implementations is detirmined by the number of recursive calls made to calculate the Fibonacci numbers. Both versions make two recursive calls for each value of $n$. This can be seen from the tail-recursive version: 'fib_tail_recursive_helper(n - 1, b, a + b)', where two calls are made. In both cases, for each value of $n$, the function makes two recursive calls. So in each implementation, each recursive call leads to two more recursive calls until the base cases are reached. Therefore the time complexity remains $O(n^2)$.
```

Comment:
There's clearly only one recursive call. You're also saying that the complexity is $2^n$ at the beginning and $n^2$ at the end. Please make sure that your solution makes sense before submitting.